menuAndAbilityBot/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "rg-unity-bossroom-menu",
+  "version": "1.0.0",
+  "description": "PlayTest bot that navigates from the main menu to the dungeon.",
```

Comment:
Should this be updated to say "navigate the menu and control the bot"?

Reply:
updated